packages/core/src/components/Layout.tsx

```diff
@@ -120,6 +152,12 @@ export default function Layout({
           onClose={onExportSuccessModalClose}
           primitives={{ colors, typography, shadows }}
         />
+        <AlertDialogDelete
+          isOpen={isAlertDialogOpen}
+          onClose={onDeleteAlertDialogClose}
+          onDelete={() => onDeleteData()}
+          deleteAllTokens={true}
+        />
```

Comment:
can you mimic the Export Settings Section and have it be in it's own "Settings"section? 

<img width="300" alt="Screen Shot 2023-04-11 at 11 28 14 PM" src="https://user-images.githubusercontent.com/29822597/231370440-807cbef3-ae07-40c8-b287-e95464884e9f.png">